docs/introduction/what_is_jellyfish.md

```diff
@@ -0,0 +1,27 @@
+---
+id: what_is_jellyfish
+slug: /
+---
+
+# What is Jellyfish?
+
+Jellyfish is an open source, general purpose media server that ships with support for multiple media protocols.
+It can be thought of as a multimedia bridge meant for creating different types of multimedia systems. 
+For example, it will be a perfect choice for a real-time video conferencing system, a broadcasting solution or both at the same time. 
+The unique feature of our media server is the ability to combine different multimedia protocols. 
+For example, one can stream video from its CCTV camera via RTSP to the server, convert it to WebRTC and send to its web application. 
```

Comment:
```suggestion
Jellyfish is an open-source, general-purpose media server that ships with support for multiple media protocols.
It can be thought of as a multimedia bridge meant for creating different types of multimedia systems. 
For example, it will be a perfect choice for a real-time video conferencing system, a broadcasting solution or both at the same time. 
The unique feature of our media server is the ability to combine different multimedia protocols. 
For example, one can stream video from its CCTV camera via RTSP to the server, convert it to WebRTC and send it to its web application. 
```